R/random-forest-development.R

```diff
@@ -3,8 +3,8 @@
 #' @description This step allows you to create a random forest model, based on
```

Comment:
Similar comment to linear mixed model... it might be nice to have one additional sentence on why it might be good to use random forest. Obviously the user can easily try all of the models but knowing that lmm is great for small row numbers where a patient is represented multiple times, random forest when data is more complex and relationship is non linear, and lasso when linear relationship is more evident/inherent in the data. *Just a thought.